src/odemis/dataio/test/csv_test.py

```diff
@@ -110,6 +110,25 @@ def testExportSpectrumNoWL(self):
             raised = True
         self.assertFalse(raised, 'Failed to read csv file')
 
+    def testExportSpectrumLine(self):
+        """Try simple spectrum-line export"""
+        size = (6, 1341)
+        dtype = numpy.float
+        data = model.DataArray(numpy.zeros(size, dtype))
```

Comment:
This is a test without MD_WL_LIST, but it should also test with MD_WL_LIST, as for the point spectrum.